GiniVision/Classes/OnboardingContainerViewController.swift

```diff
@@ -90,6 +90,15 @@ internal class OnboardingContainerViewController: UIViewController, ContainerVie
         displayContent(contentController)
     }
     
+    public override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        // Current onboarding page needs to be centered during transition (after ScrollView changes its frame)
+        coordinator.animate(alongsideTransition: { (_) in
+            (self.contentController as? OnboardingViewController)?.centerTo(page: self.pageControl.currentPage)
+        })
+        
+        super.viewWillTransition(to: size, with: coordinator)
```

Comment:
I put it before because the animation could start if super method is called before. I just checked that there is no problem, so I put the super called before in order to be consistent.